### PR TITLE
net-dns/getdns: version bump to 1.5.1

### PR DESCRIPTION
**Pull request by CaseOf**
Upstream released a new getdns version 1.5.0 ([release notes](https://getdnsapi.net/releases/getdns-1-5-0/)), also providing an updated version of stubby 0.2.4.
Then I updated my ebuild for this package.

CC @blueness

### CODE REVIEW

File: net-dns/getdns/getdns-1.5.0.ebuild [hunk 1]
```diff
@@ -29,8 +29,12 @@ DEPEND="
 	libev? ( dev-libs/libev:= )
 	libuv? ( dev-libs/libuv:= )
 "
+DEPEND="
+	${COMMON_DEPEND}
+	doc? ( app-doc/doxygen )
```
Comment (hanetzer) [inline net-dns/getdns/getdns-1.5.0.ebuild]: this (`app-doc/doxygen`) should more appropriately be in BDEPEND as it is executed
on the build host.

Comment (CaseOf) [reply net-dns/getdns/getdns-1.5.0.ebuild]: I fixed that.

File: net-dns/getdns/getdns-1.5.0.ebuild [hunk 2]
```diff
@@ -12,7 +12,7 @@ SRC_URI="https://getdnsapi.net/releases/${P//./-}/${P}.tar.gz"
 LICENSE="BSD"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE="stubby +getdns_query +getdns_server_mon libressl +idn +unbound libevent libev libuv +threads static-libs"
+IUSE="stubby +getdns_query +getdns_server_mon libressl +idn +unbound libevent libev libuv +threads static-libs doc"
```
[the remark this answers is not in the document]
Comment (CaseOf) [reply net-dns/getdns/getdns-1.5.0.ebuild]: fixed

**Comment by CaseOf** [comment]
I did updated sort again to be sure it's ok

**Comment by CaseOf** [comment]
I fixed a typo I forget while resorting items again and again, and I also bumped to 1.5.1 now that it is also released…

**Comment by CaseOf** [comment]
sorting again

**Comment by CaseOf** [comment]
ping @blueness

**Comment by CaseOf** [comment]
ping @blueness ?

**Comment by blueness** [comment]
I'm getting a failure to build:
```
checking for x86_64-pc-linux-gnu-pkg-config... /usr/bin/x86_64-pc-linux-gnu-pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for CHECK... yes
configure: Default trust anchor: /etc/unbound/getdns-root.key
configure: error: Missing dependencies: libunbound

```

**Comment by blueness** [comment]
Can I recommend that you try to build this with various combination of USE flags on a fresh stage3 chroot.  This will reveal any missing dependencies.

**Comment by blueness** [comment]
oh wait, never mind, i see it in there.  let me keep testing.

**Comment by blueness** [comment]
pushed --- also please feel free to ping me multiple times if i don't answer promptly.

**Pull request closed without merging**